cmake/Modules/FindComputeCpp.cmake

```diff
@@ -328,6 +325,10 @@ function(__build_ir)
     add_custom_target(${headerTargetName} DEPENDS ${outputDeviceFile} ${outputSyclFile})
     add_dependencies(${SDK_BUILD_IR_TARGET} ${headerTargetName})
   endif()
+  
+  if(COMPUTECPP_SDK_INCLUDE_AFTER)
```

Comment:
It's a little strange to have the SDK naming if it's in the FindComputeCpp file. I also think it makes more sense to have it as part of `add_sycl_to_target`.

That being said, I also think it's a little weird to have a large behaviour change like this exposed via a configuration option (that's why I wrote it as a per-target property in the first place). I know there are reasons why we're considering changes like this, but don't we just want to change the default? Is the idea that if a project needs include after in any file, this is a simple way of making every target use it?

Reply:
This seemed like the easiest way to achieve the desired behaviour, which yes is to be able to switch the entire project to use this property. I suppose `add_sycl_to_target` would be better, and I agree about the name. Is it possible to have an option that's named the same as a property? I don't think that should cause conflicts.